Makefile

```diff
@@ -45,11 +45,13 @@ CHART_PATH ?= kubernetes
 CHART_NAME_1 ?= cray-service
 CHART_NAME_2 ?= cray-jobs
 CHART_NAME_3 ?= cray-postgresql
-CHART_VERSION_1 ?= local
-CHART_VERSION_2 ?= local
-CHART_VERSION_3 ?= local
+CHART_VERSION_1 ?= 0.0.0
+CHART_VERSION_2 ?= 0.0.0
+CHART_VERSION_3 ?= 0.0.0
+UID := $(shell id -u)
```

Comment:
Helm-unittest `0.3.x` docker images are not run by `root`, so they needs to be run as host user to be able to write to mounted directories.